### DISCUSSION TITLE
人類是否會被生成式AI取代

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1607] [Ideas]
我認為近日AI科學家吳恩達的演講指出一個關鍵："生成式AI的潛力無窮"，其中他說"生成式AI目前的商業價值小，但未來三年價值會翻倍"，我認為近年ChatGPT的發展逐漸讓許多人擔心，人類是否終將有一天會被生成式AI取代；
但我找到這篇文章：近期IBM的究指出，隨著ChatGPT的普遍使用，帶來的是各級部門的增強，我覺得這項科技的出現使實現職位功能的強化，而非取代該位置上的工作。
該份報告總結：AI 不會取代人類，但使用 AI 的人將取代不使用 AI 的人。
我覺得這個結論不但給這個世代的科技進步一個讚賞，且也未人類的擔憂畫上一個逗號。
使我覺得不應該害怕這些科技的進步，相反的也因為有這些工具才能讓我們做得更好更快，我認為我們應該要更加增進自己，使之成為我們學習或是工作上的一大工具。
https://udn.com/news/story/6897/7393543

**Comment by v0ouo** [comment]
我認同版主的看法，現在AI越來越進步，如何正確地使用才是最重要的。像是在寫作業時，有時候會遇到許多問題，但又不知道要搜尋什麼才能知道問題的答案，這時候問AI一些問題搞不好就能得到你要的關鍵字或答案。學習如何正確使用AI並使它成為生活中的工具儼然已經成為我們現在要學習的課題。

**Comment by lazywulf** [comment]
我認同版主的說法。AI 可以成為提高生產力的工具。以 ChatGPT 為例，我們不再需要花費大量的時間在尋找資料，而是可以快速藉由 GPT 知道要如何對問題下手。或像是 copilot ，我們可以更快速完成我們想做的程式而非讀文檔讀到自己吐血。

**Comment by Yanxxx7979** [comment]
同樣認同版主的說法，並且我認為人類的創造力是不可取代的！比如最近很紅的AI繪圖出來之後，亦有人指出是畫家藝術家未來會被完全取代，我認為不是的。AI繪圖的根本上是大量對圖像特徵及規律進行訓練學習再生成，因此即便隨著算法的修正做出更精美、生成更精確的圖像，它仍是由人類給出指令而在座運算，包括現許多Youtuber也會使用AI生成的影像作為演示，也的確為我們的生活帶來更多的可能性。但我個人並不會覺得那是藝術，某種程度上我願將它類比為一組透過算法得出的數字，若與許多藝術家將自己的作品賦予意義，讓其具有藝術價值；兩者我覺得本質是完全不同且不衝突的。

**Comment by Chengbosco** [comment]
我覺得ai目前只能用來做特定任務的自動化，像整理資料，或生成簡易的圖片，所以更重要的是要學會如何和ai溝通，請它幫忙，像之前我有參加營位，是用python做x光圖片的病例分析（用反捲積迴圈什麼的），準確率還蠻高的，但ai其實也只能分類什麼照片可能發生什麼事生什麼病，並不能判斷為什麼，但作為輔助是很好的工具。

**Comment by 710388** [comment]
不會吧，感覺目前人工智慧還是沒有真正理解內涵的感覺，在很多細節上都需要人工的調整

**Comment by NcuJerryKao** [comment]
我完全同意你的觀點，並認為這是一個很有建設性的想法。以下是我的建議回覆：

生成式AI的潛力確實是無窮的，並且隨著技術的進步，它的商業價值有望在未來大幅增加[[1](https://udn.com/news/story/6897/7393543)]。

ChatGPT等技術的普及確實帶來了各個部門的增強，而非取代原有的工作。這種技術的出現使得工作更有效率，提高了部門的生產力[[1](https://udn.com/news/story/6897/7393543)]。

報告的結論指出，AI不會取代人類，但使用AI的人將取代不使用AI的人。這給了科技進步一個肯定，並緩解了人們對於被AI取代的擔憂[[1](https://udn.com/news/story/6897/7393543)]。

我們不應該害怕科技的進步，而是應該把它視為一個工具，透過學習和使用它，提升自己的技能和效率，從而更好地應對未來的學習和工作挑戰[[1](https://udn.com/news/story/6897/7393543)]。

這些建議有助於我們更好地理解和應對AI技術的發展，並將其應用於提升我們的生活和工作效率。

**Comment by Taoyi-Chiang** [comment]
我好想要被AI取代欸，讓AI或機器人去幫我工作，然後我就每天負責玩就好了。
而且取代人類，這不是本來就是AI發明來的目的嗎？我一直覺得每一個人類都值得更好的，而不只能一直加班工作╰(*°▽°*)╯
